Cargo.toml

```diff
@@ -27,15 +27,20 @@ option_if_let_else = "warn"
 all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
 
+[patch.crates-io]
+alloy-eips = { git = "https://github.com/alloy-rs/alloy", branch = "main" }
+
```

Comment:
TODO: remove this once [`BlockNumHash`](https://docs.rs/alloy-eips/latest/alloy_eips/eip1898/type.BlockNumHash.html) has serde support.